sdk/ai/azure-ai-projects/samples/evaluations/sample_model_evaluation.py

```diff
@@ -0,0 +1,119 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""
+DESCRIPTION:
+    This sample demonstrates how to create and run an evaluation for an Azure AI agent
+    using the synchronous AIProjectClient.
+
```

Comment:
The above description is the same as I see in "sample_agent_evaluation.py". Please update here or there.